tools/pontoon/workflow.md

```diff
@@ -26,8 +26,10 @@ General notes:
 2. Browse to the [project](teams_projects.md).
 3. In [filters](search_filters.md) select `Suggested`, to isolate the strings previously suggested by another translator.
```

Comment:
We should probably replace `Suggested` with `Unreviewed Suggestions` now. Also in Phase 3, Step 1.